### PR TITLE
Unflake testBookmark on TF

### PR DESCRIPTION
**Pull request by jelly**

### CODE REVIEW

**Comment by jelly** [comment]
Great success! https://artifacts.dev.testing-farm.io/1b00f39d-2644-4d10-8e00-1ae910eecc8a/

File: test/check-application [hunk 1]
```diff
@@ -1597,6 +1597,19 @@ class TestFiles(testlib.MachineCase):
...
 
+        def assert_bookmark(bookmark: str, *, exists: bool = True) -> None:
+            b.click("#bookmark-btn")
+            b.wait_visible(".pf-v5-c-menu")
+            if exists:
+                b.wait_visible(f".pf-v5-c-menu .pf-v5-c-menu__list-item:contains('{bookmark}') button")
+            else:
+                # Has to wait on file.watch() (inotify) to re-read the bookmarks
+                b.wait(lambda: bookmark not in b.text(".pf-v5-c-menu"))
+                self.assertNotIn(bookmark, b.text(".pf-v5-c-menu"))
```
Comment (martinpitt) [inline test/check-application:1608]: How is that different from `b.wait_not_in_text`?

**Review by martinpitt — DISMISSED** [review]
Thanks for tracking this down! One opportunity to make this a bit more readable, otherwise go ahead.

File: test/check-application (hunk 1 above)
Comment (jelly) [reply test/check-application]: Not much! I'll change it to our helper

**Review by martinpitt — APPROVED** [review]
Cheers!

**Pull request merged**